velox/functions/prestosql/Arithmetic.h

```diff
@@ -23,15 +23,19 @@
 #include <functional>
 #include <system_error>
 
+#include "boost/math/distributions/beta.hpp"
```

Comment:
Perhaps, start a new header file for probably functions: Probability.h

Reply:
Created new file: Probability.h